core/src/main/java/io/netty/loom/EventLoopScheduler.java

```diff
@@ -93,9 +93,16 @@ private static ThreadFactory newEventLoopSchedulerFactory(SharedRef sharedRef) {
 		// and share it between the thread attachment and the SchedulingContext,
 		// enabling
 		// work-stealing to change it for both
-		return runnable -> NettyScheduler.newEventLoopScheduledThread(() -> ScopedValue
-				.where(CURRENT_SCHEDULER, new SchedulingContext(Thread.currentThread().threadId(), sharedRef))
-				.run(runnable), sharedRef);
+		var unstartedBuilder = Thread.ofVirtual();
```

Comment:
@AlanBateman I would have expected this new method to be static in `Thread.OfVirtual`.

Second: do you think is ok to make users able to set the preferred carrier?
e.g. see the comment below